.github/workflows/linux-eic-shell.yml

```diff
@@ -149,57 +156,40 @@ jobs:
     - name: Uncompress build artifact
       run: tar -xaf build.tar.zst
     - name: Run clang-tidy on changed files
-      uses: eic/run-cvmfs-osg-eic-shell@main
       if: ${{ github.event_name == 'pull_request' }}
-      with:
-        platform-release: "${{ env.platform-release }}"
-        run: |
-          git diff ${{github.event.pull_request.base.sha}} ${{ github.event.pull_request.base.sha }} | clang-tidy-diff -p 1 -path build -quiet -export-fixes clang_tidy_fixes.yml -extra-arg='-std=c++20' -checks='-*,bugprone-*,-bugprone-narrowing-conversions' -clang-tidy-binary run-clang-tidy
+      run: |
+        git diff ${{github.event.pull_request.base.sha}} | clang-tidy-diff -p 1 -path build -quiet -export-fixes clang_tidy_fixes.yml -extra-arg='-std=c++20' -checks='-*,bugprone-*,-bugprone-narrowing-conversions' -clang-tidy-binary run-clang-tidy
```

Comment:
`${{github.event.pull_request.base.sha}}` entered twice here. I assume, it was a bug.

Reply:
Introduced 6 months ago in 17c4137878de4695286caad7e1a26982402fe626
I was wondering why we haven't had any clang-tidy reports in a long while.